src/applications/disability-benefits/view-payments/tests/components/view-payments-lists/view-payments-lists.unit.spec.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { expect } from 'chai';
+import { render } from '@testing-library/react';
+import sinon from 'sinon';
+import allPayments from '../../../reducers/index';
+import createCommonStore from 'platform/startup/store';
+
+const store = createCommonStore(allPayments);
+
+import ViewPaymentsLists from '../../../components/view-payments-lists/ViewPaymentsLists.jsx';
+
+import { payments } from '../../helpers';
+
+describe('View Payments Lists', () => {
+  it('renders View Payments Lists component', async () => {
+    const getAllPayments = sinon.spy();
+    const wrapper = render(
+      <ViewPaymentsLists
+        store={store}
+        isLoading={false}
+        error={false}
+        payments={payments}
+        getAllPayments={getAllPayments}
+      />,
+    );
+    await expect(getAllPayments.toBeCalled);
+    await expect(wrapper.findByText(/Payments you received/)).to.exist;
+    wrapper.unmount();
```

Comment:
One of the cool things about react testing library is that unmount happens automatically with mocha, so we don't need to explicitly call it. ✨ ✨ ✨  More here - https://testing-library.com/docs/react-testing-library/api#cleanup

Reply:
saweet! Updated this.